internal/executor/terminalwrapper/terminalwrapper.go

```diff
@@ -117,6 +126,14 @@ func (wrapper *Wrapper) Wait() chan Operation {
 				return
 			}
 
+			if durationSinceLastActivity >= 10 * time.Minute {
```

Comment:
Should it be `minIdleDuration` here?

Reply:
Right, I've also realized that the overall logic is a bit off, fixed in https://github.com/cirruslabs/cirrus-ci-agent/pull/192/commits/d64a9489573a16cdc6006cb2b6dd5ae547c86577.